recipes/opensearch/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "opensearch" %}
+{% set version = "2.19.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/opensearch-project/OpenSearch/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 99999a392dcf90bafebfa143ed071b45662fb022dcbcfa77df802248338d3a63
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  build:
+    - sed
+    - openjdk =21
+  host:
+    - openjdk =21
```

```suggestion
  build:
    - sed
    - openjdk =21
```
There's no need for a separate `host` in `noarch: generic` packages.